.github/workflows/main.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: [main]
```

Comment:
Did you intend to merge this change? I thought this was temporary since the PR was originally going into the other branch

Reply:
Yeah, I left it running on any pull request so we can do "stacked" PRs – not sure if there is a better way to achieve the same behaviour.

Reply:
Ah ok, that works!